src/main/flight/pid.c

```diff
@@ -579,6 +573,11 @@ void pidInitConfig(const pidProfile_t *pidProfile)
         pidCoefficient[axis].Kd = DTERM_SCALE * pidProfile->pid[axis].D;
         pidCoefficient[axis].Kf = FEEDFORWARD_SCALE * (pidProfile->pid[axis].F / 100.0f);
     }
+#ifdef USE_INTEGRATED_YAW_CONTROL
+    if (!pidProfile->use_integrated_yaw) {
+#endif        
+        pidCoefficient[FD_YAW].Ki *= 2.5f;
+    }
```

Comment:
I think this `}` must be enclosed with the ifdef USE_INTEGRATED_YAW_CONTROL too

Reply:
Or rather go

```
    if (!pidProfile->use_integrated_yaw)
#endif 
    {    
        pidCoefficient[FD_YAW].Ki *= 2.5f;
    }
```

But it's failing to build on F3 because of this now.